lib/search/query_components/core_query.rb

```diff
@@ -61,6 +61,18 @@ def quoted_phrase_query
       ]
     end
 
+    def unquoted_phrase_query
+      [
+        match_phrase("title"),
+        match_phrase("acronym"),
+        match_phrase("description"),
+        match_phrase("indexable_content"),
+        match_all_terms(%w(title acronym description indexable_content)),
```

Comment:
It's not part of this work, but I do wonder about including acronym in `match_all_terms`. I'm probably underselling it wildly, but my brain can only think of organisations that might populate this field.

Reply:
A search for `hmrc sign on` will have `match_phrase("acronym")` fail, because it's only one word (not the full phrase) in the acronym; but will match the `match_all_terms` (assuming "sign on" is in the page somewhere)